antora-ui-camel/src/css/frontpage.css

```
@@ -197,12 +197,14 @@ section.frontpage h2 {
 }
 
 .frontpage .box {
-  flex: 50%;
-  padding: 1.2rem 0;
+  flex: 33%;
+  flex-direction: column;
+  margin: 1rem;
+  padding: 1.2rem 0.75rem;
+  border: 1px solid #e1e1e1;
```

We need to use variables

```suggestion
  border: 1px solid var(--color-smoke-90);
```